src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,44 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "cinema_hall")
```

Comment:
minor, but in `Movie` class table name is defined as plural. Although it is discussible, please be consistent with those approach

Reply:
fixed in Movie class. I usually name tables in a singular form, just didn't want to change the code I got from git. Fixed in the Movie class.